external/CMakeLists.txt

```diff
@@ -102,3 +102,15 @@ if (NOT ${SPIRV_SKIP_TESTS})
     endif()
   endif()
 endif()
+
+if(SPIRV_BUILD_FUZZER)
+  set(PROTOBUF_DIR ${CMAKE_CURRENT_SOURCE_DIR}/protobuf/cmake)
+  set(protobuf_BUILD_TESTS OFF CACHE BOOL "Disable protobuf tests")
```

Comment:
these protobufs are lower case? weird.

Reply:
Weird indeed but yes, they are - thanks for approving!